core/src/main/java/de/danoeh/antennapod/core/service/playback/PlaybackService.java

```diff
@@ -1213,13 +1213,14 @@ private synchronized void setupNotification(final Playable playable) {
         if (notificationSetupThread != null) {
             notificationSetupThread.interrupt();
         }
+        if (!started) {
+            stopService();
```

Comment:
Shouldn't there be a `return;` statement after `stopService()`? I am unclear on Android's behavior if `return` is not there.